google/cloud/CMakeLists.txt

```diff
@@ -264,8 +264,21 @@ install(EXPORT google_cloud_cpp_common-targets
 install(TARGETS google_cloud_cpp_common google_cloud_cpp_common_options
         EXPORT google_cloud_cpp_common-targets
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+                COMPONENT google_cloud_cpp_runtime
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+                COMPONENT google_cloud_cpp_spanner_runtime
```

Comment:
Here and there, I guess it should be `google_cloud_cpp_runtime`.

Reply:
Good catch! thanks.